i18n.md

```diff
@@ -383,3 +390,15 @@ $ gulp available-languages
 $ git commit client/gettext/po client/gettext/json
 $ git push
 ```
+
+
+## Fast-forwarding translations from release branch to master branch
+
+Ordinarilly, translations for ManageIQ are done for particular release and the strings for translations are therefore taken from a release branch. When the translation work for the release branch is done, it is a good practice to fast-forward the translations from the release branch to the master branch. One way to do the fast-forward is to utilize Transifex. The following example assumes fast forward from jansa branch to master:
```

Comment:
typo: `Ordinarilly` should be `Ordinarily`

Reply:
Typo fixed.